desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -205,7 +205,7 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
         <div>
           <div class="control-group" data-bind="visible: ((createWizard.prefill.target_type().length == 0 || createWizard.prefill.source_type() == 'all') && createWizard.source.inputFormats().length > 1)">
             <label for="sourceType" class="control-label"><div>${ _('Type') }</div>
-              <select id="sourceType" data-bind="selectize: createWizard.source.inputFormats, value: createWizard.source.inputFormat, optionsText: 'name', optionsValue: 'value'"></select>
+              <select data-hue-analytics="importer:test" id="sourceType" data-bind="selectize: createWizard.source.inputFormats, value: createWizard.source.inputFormat, optionsText: 'name', optionsValue: 'value'"></select>
```

Comment:
"test"? Should it not be importer:select-input-format? Or perhaps only log on a subscribe of createWizard.source.inputFormat..